packages/cells/src/cells/date-picker-cell.tsx

```diff
@@ -18,49 +115,29 @@ const renderer: CustomRenderer<DatePickerCell> = {
         drawTextCell(args, displayDate, cell.contentAlign);
         return true;
     },
-    // eslint-disable-next-line react/display-name
-    provideEditor: () => p => {
-        const cellData = p.value.data;
-        const { format, date } = cellData;
+    provideEditor: () => ({
+        editor: Editor,
+    }),
+    onPaste: (v, d) => {
+        let parseDateTimestamp: number = NaN;
+        // We only try to parse the value if it is not empty/undefined/null:
+        if (v) {
+            // Support for unix timestamps (milliseconds since 1970-01-01):
+            parseDateTimestamp = new Number(v).valueOf();
 
-        let val = "";
-        if (date !== undefined) {
-            val = date.toISOString();
-            if (format === "date") {
-                val = val.split("T")[0];
-            } else {
-                val = val.substring(0, 23);
+            if (Number.isNaN(parseDateTimestamp)) {
+                // Support for parsing ISO 8601 date strings:
+                parseDateTimestamp = Date.parse(v);
+                if (d.format === "time" && Number.isNaN(parseDateTimestamp)) {
+                    // The pasted value was not a valid date string
+                    // Try to interpret value as time string instead (HH:mm:ss)
+                    parseDateTimestamp = Date.parse(`1970-01-01T${v}Z`);
```

Comment:
Why not interpret it as "today" instead of 1970? Today at least feels more likely to be expected.

Reply:
I assume we could also use today's date here...  both are probably semi-optimal solutions here. The general problem here is that there isn't really a native format for time in javascript.

Reply:
thinking a bit more here. Probably today has slightly more advantages 👍 @willhuang1997 would you be fine as well with changing that?

Reply:
One argument for keeping `1970-01-01`: It is consistent with how the time input converts the time back into a date. Calling `valueAsDate` on the time input event will return the time based on `1970-01-01`. `valueAsNumber` is also the UNIX timestamp from `1970-01-01` and not today.

<img width="928" alt="image" src="https://user-images.githubusercontent.com/2852129/218188270-4842a7da-fc0c-40aa-8200-480adabbeb7b.png">